docs/application/native/guides/connectivity/connectivity-cover-n.md

```diff
@@ -41,4 +44,4 @@ You can allow the users to initialize the VPN device, and manage routing, DNS, a
 You can use IoTivity features in Tizen. You can use the seamless device-to-device connectivity to address the needs of the Internet of Things (IoT) through the open source reference implementation of the OIC (Open Interconnect Consortium) standard specifications.
```

Comment:
Use IoT Activity feature in Tizen and also seamless device-to-device connectivity to address the needs of the Internet of Things (IoT) through the open source reference implementation of the OIC (Open Interconnect Consortium) standard specifications.